phpBB/posting.php

```diff
@@ -1238,6 +1238,7 @@
 	*
 	* @event core.posting_modify_submission_errors
 	* @var	array	post_data	Array with post data
+	* @var	array	poll		Array with poll data from post (must be used instead of the post_data equivalent)
```

Comment:
Ok, forgot it here, while I did it in the below case.  Will send a new commit with these changes.